.env

```diff
@@ -53,13 +55,13 @@ _APP_STORAGE_PREVIEW_LIMIT=20000000
 _APP_FUNCTIONS_SIZE_LIMIT=30000000
 _APP_FUNCTIONS_TIMEOUT=900
 _APP_FUNCTIONS_BUILD_TIMEOUT=900
-_APP_FUNCTIONS_CPUS=1
-_APP_FUNCTIONS_MEMORY=512
+_APP_FUNCTIONS_CPUS=4
+_APP_FUNCTIONS_MEMORY=2048
```

Comment:
Do we need this increased amount for testing as well ?

Reply:
Just for development is fine. We can probably do a bit lower. I think it was 512 before, and java/dart builds are failing with that - they run out of memory.
We can set CPU to 1, that doesn't matter too much